pkg/commands/artifact/run.go

```diff
@@ -250,6 +250,7 @@ func (r *runner) Filter(ctx context.Context, opts flag.Options, report types.Rep
 func (r *runner) Report(opts flag.Options, report types.Report) error {
 	if err := pkgReport.Write(report, pkgReport.Option{
 		AppVersion:         opts.AppVersion,
+		DfctlVersion:       opts.DfctlVersion,
```

Comment:
This still needs a change to set dfctl version, can be passed internally from dfctl - should be doable in case of CLI
But what in case of webhook?

Reply:
In case of webhook also, since we will use dfctl, won't this attribute be available?

Reply:
ohh I was not aware of the move from dfscan to dfctl. This is ok in that case
Can you create a ticket to pass version from dfctl to dfscan?

Reply:
I think Monika created a ticket for the same. https://df-eng.atlassian.net/browse/DEEP-6845
Once we pass dfctl version to dfscan and use it during scan registration, instead of hardcoded value,  this bug will automatically get resolved.